SD-K9/src/app/components/building-info/building-info.component.ts

```diff
@@ -102,7 +113,9 @@ export class BuildingInfoComponent implements OnInit, OnDestroy {
 
   hideBuildingInfoCard(event) {
     this.buildingInfoCardIsShown = false;
-    this._buildingInfoStore.clearBuildingInfo();
+
+    //Makes clickable buildings more reliable
+    //this._buildingInfoStore.clearBuildingInfo();
```

Comment:
Should delete if not useful

Reply:
I'll uncomment it since it's to makde hideBuildingInfoCard for reliable